### DISCUSSION TITLE
No imaging data or sessions found

### DISCUSSION THREAD

**Discussion by jasewe** [QSMxT/QSMxT#120] [Q&A] [answered]
Hello everyone, and thank you very much for providing the software. I am quite new to the field of MRI data processing. This is probably why I am having trouble running the qsmxt command. I have converted my REC data to Nifti data outside of the program using MRIcrone. Within the qsmxt software, I then converted my data to be BIDS-compliant:

sub-05/
    ├── anat/
    │   ├── sub-05_echo-1_part-mag.nii
    │   ├── sub-05_echo-1_part-mag.json
    │   ├── sub-05_echo-1_part-phase.nii
    │   ├── sub-05_echo-1_part-phase.json
    │   ├── sub-05_echo-2_part-mag.nii
    │   ├── sub-05_echo-2_part-mag.json
    │   ├── sub-05_echo-2_part-phase.nii
    │  ....

The files are not corrupted and are complete. For each of the five echo times, I have a magnitude file and a phase file. The .json files are also complete. However, when I enter the qsmxt command, I receive the following message:

[WARNING]: No imaging data or sessions found in /home/jovyan/Desktop/bids/sub-05/anat/sub-05_echo-e1_part-mag.nii 

This message appears for all the files. The data seems to be unreadable. Does anyone have an idea why this might be?

Thanks in advance!! :-)

**Comment by astewartau** [comment, marked as answer]
Hi, and thank you for the question! 

At first glance, the BIDS structure you posted seems to not be compliant, as the files are missing an appropriate suffix, which for multi-echo data will typically be "MEGRE". 

If you are starting from DICOMs and using `dicom-convert`, this suffix should be automatically added. If you are starting from NIfTI images and using `nifti-convert`, you will need to fill "MEGRE" into the suffix field on the conversion spreadsheet. 

You can see some examples of compliant BIDS structures here: https://qsmxt.github.io/QSMxT/using-qsmxt/data-preparation#bids-examples

**Reply by jasewe** [reply]
Thanks!! This actually worked :-)

**Reply by astewartau** [reply]
Great! Please don't hesitate to post if you have more questions. 😁
